### ISSUE TITLE
Flowly SVGs should be hidden

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#104]
## Description
I think we discussed this over slack, but just in case I'm dropping it as a ticket ⭐ 
We'll just want to make sure these have `aria-hidden="true"` so they don't sneak up on folks like this

<img width="936" alt="Screen Shot 2022-09-23 at 6 33 40 PM" src="https://user-images.githubusercontent.com/14154792/192065424-b2ace517-6c40-4bbe-aae6-8f16bff49126.png">

## Success criteria
- [ ] Decorative svgs have the attribute `aria-hidden="true"`
